backend/app/services/channels/dingtalk/user_mapping.py

```
@@ -0,0 +1,133 @@
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from typing import Optional
+
+__all__ = [
+    "MappedUserInfo",
+    "BaseUserMapper",
+    "DefaultUserMapper",
+    "ERPUserMapper",
+    "get_user_mapper",
+    "set_user_mapper",
+]
```

Ruff (0.14.14)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)